ts/addon.ts

```diff
@@ -95,11 +100,15 @@ export default addon({
     return !['in-repo-a', 'in-repo-b'].includes(addon.name);
   },
 
-  _checkBabelVersion() {
+  _getBabelVersion() {
     let babel = this.parent.addons.find(addon => addon.name === 'ember-cli-babel');
-    let version = babel && babel.pkg.version;
-    if (!babel || !(semver.gte(version!, '7.1.0') && semver.lt(version!, '8.0.0'))) {
-      let versionString = babel ? `version ${babel.pkg.version}` : `no instance of ember-cli-babel`;
+    return babel && babel.pkg.version;
+  },
+
+  _checkBabelVersion() {
+    let version = this._getBabelVersion();
+    if (!version || !(semver.gte(version!, '7.1.0') && semver.lt(version!, '8.0.0'))) {
```

Comment:
```suggestion
    if (!version || !(semver.gte(version, '7.1.0') && semver.lt(version, '8.0.0'))) {
```
Non-null assertions aren't necessary here (and in general are only advisable tests). On the line before, version is `string | undefined`, and your first condition narrows the `undefined` away from the remaining two